### PR TITLE
powerplatform-vscode Loc checkin for WO71206

### PR DESCRIPTION
**Pull request by v-hifuku-msft**
powerplatform-vscode Loc checkin for WO71206

### CODE REVIEW

File: loc/translations-import/vscode-powerplatform.fr.xlf [hunk 1]
```diff
@@ -145,6 +145,46 @@ L’authentification interactive n’est pas disponible pour les scénarios dist
         <source xml:lang="en">PAC CLI is not supported in VS Code for the Web</source>
         <target state="translated">PAC CLI n’est pas pris en charge dans VS Code pour le Web</target>
       </trans-unit>
+      <trans-unit id="microsoft-powerapps-portals.walkthrough.title">
```
Comment (tehcrashxor) [inline loc/translations-import/vscode-powerplatform.fr.xlf:148]: Looks like there are a decent number of strings in the `package.json` section of the translated xlf files here that don't have translations added.

I _think_ that should be fine, as these are used by the Web Only version of the extension, which currently doesn't load translations anyway due to a VS Code Web issue.   @ashishchoudhary001 , can you confirm that?

**Review by tehcrashxor — APPROVED** [review]
Spot checked a couple languages, and the strings used by the main version of the extension are loading as expected.   Checking with @ashishchoudhary001 about the missing translations used by the Web extension per the other comment.

**Comment by v-hifuku-msft** [comment]
Please note that additional stings were added later last week. We are handing them to localization today. ETA is this Thursday.

Thanks,
Hiro

File: loc/translations-import/vscode-powerplatform.fr.xlf (hunk 1 above)
Comment (tehcrashxor) [reply loc/translations-import/vscode-powerplatform.fr.xlf:148]: Resolving comment per Hiro's reply below - these strings will be localized later this week.

**Pull request merged**